### PR TITLE
MOVE_COMMAND: Add failure condition for bad units

### PR DESCRIPTION
**Pull request by LucasBA**

### CODE REVIEW

**Review by RustyBamboo — APPROVED** [review]
Looks good.

File: command/sub8_missions/nodes/move_command [hunk 1]
```diff
@@ -30,7 +30,6 @@ def main(args):
     commands = args.commands
     arguments = commands[1::2] #Split into commands and arguments, every other index
     commands = commands[0::2]
-
```
Comment (RustyBamboo) [inline command/sub8_missions/nodes/move_command:33]: Not sure if this was intentional

**Comment by LucasBA** [comment]
I am confused about what is causing the checks to fail, it looks like there is an issue with alarms but I didn't change anything related to alarms.

**Review by kev-the-dev — APPROVED** [review]
Thanks

**Pull request merged**